src/utils/add-due-date-comment.ts

```diff
@@ -0,0 +1,18 @@
+import { CommentList, SingleComment } from '../types/github';
+import { paginateAllCommentsOnIssue } from './paginate-comments-on-issue';
+import { context } from '@actions/github';
+import { octokit } from '../octokit';
+
+export const addDueDateComment = async (deadline: number, createdDate: Date, issue_number: number, hasExistingComments: boolean) => {
```

Comment:
Instead of passing a boolean as an argument, let's pass the `comments` array and evaluate the logic inside here

Reply:
`comments` in this case is just an integer representing the number of comments, so we'd still need to retrieve the comment list from the API. do you still want me to move the boolean check into the function?

Reply:
Yes, sorry didn't realize it was a number